### ISSUE TITLE
Incorrect number of kits in inventory

### ISSUE THREAD

**Issue by Lord-Spock** [dsiekiera/foe-helfer-extension#827]
Hi
The overview of kits in inventory shows incorrect numbers.
It  seems it shows the amount of buildings of the highest age. There can be multiple inventory items for the same building cause of different ages.

**Comment by andrgin** [comment]
Can you post a screenshot please

**Comment by Lord-Spock** [comment]
![image](https://user-images.githubusercontent.com/63965444/79691651-30c6e080-8261-11ea-8f58-09c1e2336302.png)

![image](https://user-images.githubusercontent.com/63965444/79691665-41775680-8261-11ea-8e70-aeea0a5f30eb.png)

**Comment by andrgin** [comment]
I think building kits are detected when starting the game. Can you check if you get the correct numbers after reloading the game?

**Comment by denisebuilder** [comment]
additionally to the above; the upgrade kits for some buildings are not shown or not shown next to the initial buildings

**Comment by denisebuilder** [comment]
![image](https://user-images.githubusercontent.com/64001765/79727302-7ed4f600-82ec-11ea-8b62-244f1afced95.png)
![image](https://user-images.githubusercontent.com/64001765/79727344-901e0280-82ec-11ea-9525-f07ce5898526.png)
![image](https://user-images.githubusercontent.com/64001765/79727412-a4fa9600-82ec-11ea-8031-b36f799cfe57.png)

**Comment by Lord-Spock** [comment]
With starting the game, you just mean to refresh the site ??
That is not it. Some of the items (like the entrances) are spread over 3 ages.

**Comment by Lord-Spock** [comment]
You are correct that the overview is from when you load the game from a placing/using perspective. When I place something from inventory, the overview is not updated before reloading the game.
Nevertheless, that is not the root cause of the issue I reported.

**Comment by dsiekiera** [comment]
I know what u mean and will fix it.

**Comment by Lord-Spock** [comment]
Super ! You are the best.

**Comment by Parama91** [comment]
The inventory is compared to the JSON (https://cache.foe-helper.com/kits/sets.json) and only if found in the JSON it will be shown. So you have to expand the JSON.

Code: kits.js - line 34 loading the JSON, line 108 walking throught the JSON. So if the entries are missing in the JSON, there will be no match -> so no entry
